VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1147,6 +1151,10 @@ export abstract class WaitingRoomBaseDirective {
         return this.participant.role === Role.Judge || this.participant.role === Role.StaffMember;
     }
 
+    isAHostInHearing(participants: ParticipantResponse[]): boolean {
```

Comment:
Sorry, might be my fault with how I worded the request before but now that it's an array being passed in I think this would be better as hasAHostInHearing.